### DISCUSSION TITLE
对于本地谱面管理的建议

### DISCUSSION THREAD

**Discussion by starsky919** [TeamFlos/phira#235] [Ideas]
希望能对本地谱面进行排序（按照添加时间、游玩时间、游玩次数等）以及批量删除。

**Comment by syesjohnny** [comment]
還有自訂合集和尋找

**Comment by Wooffee** [comment]
It's no more than a tiny nitpick of mine, but I do love some small quality of life changes, like adding bulk download/delete charts
We'll appreciate that if you can add it, thanks in advance

**Comment by sjfhsjfh** [comment]
功能进度追踪:

- [x] 搜索 #505
- [ ] 排序
- [ ] 批量删除

**Comment by YuevUwU** [comment]
### Checklist

- [x] Custom Collections ([phira.moe](https://phira.moe/collection) upload available)
- [x] Local Search (search by id available)
- [x] Bulk Delete
- [ ] Bulk Download
- [ ] More Sorting Method
  - [ ] Download/Import Date (Also mentioned in #555)
  - [ ] Recent Play
  - [ ] Play Count
  - [x] Difficulty
  - [ ] Chart Size (from #555)
- [ ] Local Filter

**Comment by YuevUwU** [comment]
或许还可以来个 本地 Filter
